ansible/roles/jenkins-worker/templates/aix.rc2.j2

```diff
@@ -17,7 +17,7 @@ start )
               export NODE_TEST_DIR=$HOME/tmp; \
               export NODE_COMMON_PIPE="$HOME/test.pipe"; \
               export OSTYPE=AIX72; \
-              export JOBS={{ jobs_env }}; \
+              export JOBS={{ server_jobs }}; \
```

Comment:
You can leave this as `jobs_env` and remove https://github.com/nodejs/build/blob/f7de7bfc85736a93062740629d5f1da42270ee53/ansible/roles/jenkins-worker/vars/main.yml#L144 as `jobs_env` is set from `server_jobs` if not set in `jobs_variants`: https://github.com/nodejs/build/blob/f7de7bfc85736a93062740629d5f1da42270ee53/ansible/roles/jenkins-worker/tasks/main.yml#L7-L25

Reply:
did not know that pushing up the change now